spree_backend.gemspec

```diff
@@ -25,21 +25,23 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
   s.requirements << 'none'
 
-  s.add_dependency 'spree', ">= 4.4.0.rc1"
+  s.add_dependency 'spree', ">= 4.4.0"
 
   s.add_dependency 'babel-transpiler', '~> 0.7'
   s.add_dependency 'bootstrap',        '~> 4.0'
-  s.add_dependency 'glyphicons',       '~> 1.0'
-  s.add_dependency 'flatpickr',        '~> 4.6'
   s.add_dependency 'flag-icons-rails', '~> 3.4'
+  s.add_dependency 'flatpickr', '~> 4.6'
+  s.add_dependency 'glyphicons', '~> 1.0'
   s.add_dependency 'hotwire-rails'
-  s.add_dependency 'inline_svg',       '~> 1.5'
+  s.add_dependency 'inline_svg', '~> 1.5'
+  s.add_dependency 'jquery-rails', '~> 4.3'
+  s.add_dependency 'jquery-ui-rails', '~> 6.0'
   s.add_dependency 'jsbundling-rails'
-  s.add_dependency 'jquery-rails',     '~> 4.3'
-  s.add_dependency 'jquery-ui-rails',  '~> 6.0'
+  s.add_dependency 'paranoia'
+  s.add_dependency 'ransack'
```

Comment:
@MatthewKennedy this gems are declared as deps in spree core, no need to add them here

Reply:
On it...